src/runtime/lib/utils/profileHelper.ts

```diff
@@ -144,9 +138,13 @@ function selectVendorId(vendors: Array<model.v1.vendorManagement.Vendor>, contex
 }
 
 export function resolveVendorId(profileName: string): string {
-    let vendorId;
+    let vendorId: string;
     if (profileName === AUTH.PLACEHOLDER_ENVIRONMENT_VAR_PROFILE_NAME) {
+        if (process.env.ASK_VENDOR_ID === undefined) {
+            throw new Error('Please make sure env var ASK_VENDOR_ID is defined.');
```

Comment:
Is this being handled in all the methods that are calling this function? We should also raise an `AskError` with proper logging if this leads to a failure in toolkit operations.

Reply:
Ummm, cause I saw this function already have branch to throw error: https://github.com/alexa/ask-toolkit-for-vscode/blob/master/src/runtime/lib/utils/profileHelper.ts#L156, thus I haven't checked carefully at caller side. After checking, seems not all caller side will throw AskError, will try to fix them.